src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
hmm, I'd love it if we could get away with not using `SketchX`.

The exported API can be split into "things that fit into react-primitives" (View, Text, Image etc) and "Sketch-specific concepts" — TextStyles, Artboard etc.

I agree that `Symbol` is not the right name for it because of clashes, but what if we just exported the `symbolize` or `makeSymbol` or whatever HOC rather than `Symbol.symbolize`? I guess we also need `Symbol.inject` but there's probably a smart way around that 🤔